src/locales/nl-SR.test.ts

```diff
@@ -0,0 +1,235 @@
+import { cloneDeep } from 'lodash';
```

Comment:
Please move the test file inside `__tests__` directory.

Reply:
yes I will move it.